### PR TITLE
lcm_

### LINKED ISSUE #21625: lcm_
Todo list: #3612

### PR DESCRIPTION
**Pull request by NiteshK84**
Closes #21625
Corresponding to PR: #21626

### CODE REVIEW

**Comment by NiteshK84** [comment]
There are 2 new lint test failing from other contributions:
image
image
image


I have fixed these since they didn't belong to me, but for sure they will cause the CI lint test to fail, please tell me weather should i fix theme or not?

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -1800,6 +1800,24 @@ def isnan(self):
     def lcm(self, other, *, out=None):
         return torch_frontend.lcm(self, other, out=out)
```
Comment (umairjavaid) [inline ivy/functional/frontends/torch/tensor.py:1801]: torch frontend tensor already has lcm implemented, isn't this redundant?

**Comment by umairjavaid** [comment]
> There are 2 new lint test failing from other contributions: image
> 
> First lint image
> 
> Another image
> 
> I have not fixed these since they didn't belong to me, but for sure they will cause the CI lint test to fail, please tell me weather should i fix theme or not?

If you could fix them that would be great! if not, that is still alright

**Review by umairjavaid — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/torch/tensor.py (hunk 1 above)
Comment (NiteshK84) [reply ivy/functional/frontends/torch/tensor.py:1801]: actually this is the inplace version of the lcm function, you can check that in the torch docs.

**Review by umairjavaid — CHANGES_REQUESTED** [review]
The intelligent tests are not passing. Getting the ```ModuleNotFoundError: No module named 'dill'```. This was fixed a few days ago. Could you please update your fork by running this command ./merge_with_master tf_frontend, assuming u have the ivy main set as your upstream.

**Comment by NiteshK84** [comment]
Hey @umairjavaid, 
I have fixed the dill error, 
please let me know of any left issues.
Thank you for your time.

**Comment by NiteshK84** [comment]
@umairjavaid, i am ready for the review.
Thanks

**Comment by umairjavaid** [comment]
great work :)

**Pull request merged**